utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.android.util.platformparameter
+
+/**
+ * This file contains all the constants that are associated with individual Platform Parameters.
+ * These constants are:
+ *  - Qualifier Annotation
+ *  - Platform Parameter Name
+ *  - Platform Parameter Default Value
+ * */
```

Comment:
I'd add the test constants you used in `FakePlatformParameterModule` and then getting the constants form this file so that it is more aligned with prod code.

Reply:
Okay, Done

Reply:
+1 I still see the parameters here. Can we move them into the fake module?

Reply:
Oh I was actually saying move them from the test module to this file since in prod we're retrieving parameters and values from this constants file.

What we probably really want to do is add a `TestPlatformParameterConstants` file for these test values or have them in the test file directly. @vinitamurthi WDYT?

Reply:
Since we only need to use these constants wherever the FakeModule is being used. So we can either 
- Shift these constants to FakeModule or
- Include both the Fake Module and constants inside the TestClass because we won't test the Fake Module anywhere else in Future I guess

Reply:
I am okay with either approach as long as we move it out of this file because if I understand correctly, this is in the util module which will be included in the prod build. But this seems to be only for tests, so we should move it to  `utility/src/test/...` or in `testing/...`
Hope this makes sense!

Reply:
Agree, don't want in prod. I see it in the test module so that's fine for it!